vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -365,8 +365,8 @@ struct CompTextHost : public winrt::implements<CompTextHost, ITextHost> {
 
   //@cmember Get the character to display for password input
   HRESULT TxGetPasswordChar(_Out_ TCHAR *pch) override {
```

Comment:
please don't use TCHAR, all our code should be unicode so you can use the native wchar_t type (or `std::` types)